src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -32,16 +32,28 @@
  * @see <a href="https://gist.github.com/rherrmann/7447571">Starting point</a>
  */
 public class ConditionalIgnoreRule implements MethodRule {
+	private static <A extends Annotation> Stream<A> annotation(Annotatable source, Class<A> type) {
+		return Stream
```

Comment:
\[Checkstyle\] WARNING: 'method def' child has incorrect indentation level 16, expected level should be 4\.